anet.yml

```diff
@@ -214,29 +214,71 @@ dictionary:
         style:
           height: 400px
       topLevel:
-         shortLabel: Objective
-         shortName:
-           label: Objective name
-           placeholder: Enter an objective name, example....
-         longLabel: Objectives
-         longName:
-           label: Objective description
-           placeholder: Enter an objective description, example ....
-           componentClass: textarea
-           style:
-             height: 400px
+        shortLabel: Objective
+        shortName:
+          label: Objective name
+          placeholder: Enter an objective name, example....
+        longLabel: Objectives
+        longName:
+          label: Objective description
+          placeholder: Enter an objective description, example ....
+          componentClass: textarea
+          style:
+            height: 400px
+        assessment:
+          customFields:
```

Comment:
@VassilIordanov I would think customFields doesn't add anything here and would just put the customFields directly under assessment. Or did you put it here to make it clear that the items have to follow the customFields schema?

Reply:
I was thinking maybe of adding another sibling to `customFields:` such as `assessmentPeriod` as weekly/bi-weekly/monthly/bi-monthly/yearly... etc

Reply:
Than maybe "questions" would be better a better key name than "customFields". CustomFields are just the fields of an entity which are ANET instance specific.

Reply:
good point!

Reply:
It's better to do the renaming as part of https://github.com/NCI-Agency/anet/tree/GH-2954.
We will than do it the same for persons.